static/app/views/settings/account/notifications/fields2.tsx

```diff
@@ -0,0 +1,52 @@
+import {t} from 'app/locale';
+
+export type NotificationSettingField = {
+  name: string;
+  type: 'select';
+  label: string;
+  choices?: string[][];
+  defaultValue?: string;
+  defaultFieldName?: string;
+};
+
+export const NOTIFICATION_SETTING_FIELDS: Record<string, NotificationSettingField> = {
```

Comment:
I'd prefer to keep these in separate files for now to avoid mixing them up. Once we've implemented project search and put the provider selector behind a feature flag, this page will replace the old one.